guides/common/modules/ref_supported-operating-systems.adoc

```diff
@@ -3,8 +3,8 @@
 
 ifdef::satellite[]
 You can install the operating system from a disc, local ISO image, kickstart, or any other method that Red{nbsp}Hat supports.
-Red{nbsp}Hat {ProductName} is supported only on the latest versions of {RHEL} 7 Server that is available at the time when {ProductName} is installed.
-Previous versions of {RHEL} including EUS or z-stream are not supported.
+Red{nbsp}Hat {ProductName} is supported on the latest versions of {RHEL} 8 Server, and {RHEL} 7 Server that are available at the time when {ProductName} is installed.
```

Comment:
```suggestion
Red{nbsp}Hat {ProductName} is supported on the latest versions of {RHEL} 8, and {RHEL} 7 Server that are available at the time when {ProductName} is installed.
```

RHEL8 has no difference between Server and Workstation anymore

Reply:
Resolved.